posts/2025-02-17_theres_a_pharmaverse_package_for_that/managing-clinical-trial-data.qmd

```diff
@@ -0,0 +1,323 @@
+
+Whether you’re collecting, validating, analyzing, or preparing data for regulatory submission, there’s a pharmaverse package designed to support your workflow and help you work smarter.
+
+This post covers:
+
+-   Key stages of clinical trials and the R packages that support them
+
+-   Getting started with metadata-driven clinical reporting
+
+-   The process from raw data to ADaMs, including validation
+
+-   Creating ADSL datasets and essential programming steps
+
+-   Key players in Pharmaverse and whether you need all packages
+
+-   How pharmaverse compares to Tidyverse and how to learn it
+
+By the end, you'll have a clear understanding of how Pharmaverse supports clinical trial operations and how to apply these tools in your work.
+
+## **Key Stages of the Clinical Trial Process**
+
+Managing clinical trial data involves multiple stages, each with its own challenges. pharmaverse provides a range of packages for different parts of the process, sometimes even more than one option for the same task, so you can choose what works best for your needs rather than sticking to a single approach
+
+![](images/clinical-trial-process.png)
+
+1.  **Collection** – Initial data handling using:
+
+    -   [**tidyverse**](https://www.tidyverse.org/) for general data manipulation.
+
+    -   [**diffdf**](https://cran.r-project.org/web/packages/diffdf/vignettes/diffdf-basic.html) for tracking differences in datasets.
+
+    -   [**metatools**](https://pharmaverse.github.io/metatools/) for metadata management and transformation.
+
+2.  **Tabulation** – Structuring and validating data with:
+
+    -   [**thevalidatoR**](https://github.com/marketplace/actions/r-package-validation-report) for data validation checks.
+
+    -   [**covtracer**](https://github.com/Genentech/covtracer) for ensuring code traceability and reproducibility.
+
+3.  **Analysis Ready** – Preparing data for regulatory-grade analytics:
+
+    -   [**admiral**](https://github.com/pharmaverse/admiral) for standardized data derivations.
+
+    -   [**metacore**](https://cran.r-project.org/web/packages/metacore/index.html) for metadata-driven structures.
+
+    -   [**dplyr**](https://dplyr.tidyverse.org/) (part of the tidyverse) for data manipulation
+
+4.  **Analysis Results** – Incorporating visualization and modeling tools:
+
+    -   [**ggplot2**](https://ggplot2.tidyverse.org/) (part of the tidyverse) for advanced data visualization.
+
+    -   [**visR**](https://github.com/openpharma/visR) for survival analysis and Kaplan-Meier plots.
+
+    -   [**pharmaRTF**](https://cran.r-project.org/web/packages/pharmaRTF/index.html) for creating submission-ready reports.
+
+    -   [**rtables**](https://github.com/insightsengineering/rtables) for structured table outputs
```

Comment:
I would remove rtables here if mentioned below

Reply:
also could move pharmaRTF and Tplyr down to that list of table packages rather than having their own bullets